examples/custom-service-account/README.md

```diff
@@ -22,8 +24,6 @@ provider "lacework" {}
 module "lacework_svc_account" {
   source               = "lacework/service-account/gcp"
   version              = "~> 0.1.0"
```

Comment:
All versions should be prepared for the major version bump we are going to do:
```suggestion
  version              = "~> 1.0"
```

Reply:
For doubts about why we are following this new version pin, read https://lacework.atlassian.net/wiki/spaces/ALLY/pages/1844511095/Semantic+Versioning